packages/@react-spectrum/calendar/src/CalendarTableHeader.tsx

```diff
@@ -38,12 +35,12 @@ export function CalendarTableHeader({weekStart}: CalendarTableHeaderProps) {
             return (
               <th
                 key={index}
-                aria-colindex={index + 1}
-                {...columnHeaderProps}
                 className={classNames(styles, 'spectrum-Calendar-tableCell')}>
-                <abbr className={classNames(styles, 'spectrum-Calendar-dayOfWeek')} title={dayLong}>
```

Comment:
So, I did some testing. VoiceOver doesn't seem to announce this in Safari at all, and it announces strangely as a group in Chrome. Plus, I'm not sure Spectrum wants a visible tooltip on there. I assume you were trying to get screen readers to announce the full day name rather than the abbreviation. I tried a bunch of things and the best I could come up with is the below solution of using a VisuallyHidden for the full day name, and making the abbreviation aria-hidden. Let me know what you think.

Reply:
Your approach seems to work well, and avoids the double announcement the day. It's unfortunate that the appropriate HTML element for the task doesn't work as well.